client/src/urlConfig.js

```diff
@@ -0,0 +1 @@
+export const api = "http://localhost:5000/api";
```

Comment:
declare in ENV file and update env.example

Reply:
ok